### ISSUE TITLE
check openmp cmake usage for consistency   

### ISSUE THREAD

**Issue by cyrush** [Alpine-DAV/ascent#335]
issue so I don't forget to look at this in the future. 

in ascent lib cmake lists:
```
if(ASCENT_VTKM_ENABLED)
    set(ASCENT_VTKM_USE_CUDA ${CUDA_FOUND})
    set(ASCENT_VTKM_USE_OPENMP ${ENABLE_OPENMP})
endif()


set(ASCENT_USE_OPENMP  ${OPENMP_FOUND})
```

**Comment by cyrush** [comment]
we should be using `OPENMP_FOUND`, I'll audit soon.
There are places where we use ZZZ_ENABLE_OPENMP in compiler defines -- that's fine. Just want to be consistent about our CMake If tests

**Comment by cyrush** [comment]
resolved in #346
